lib/src/beacon_settings.dart

```diff
@@ -190,6 +190,76 @@ extension SettingsX on Settings {
     );
     return s;
   }
+
+  /// Creates a new [Setting] that manages an [int].
+  ///
+  /// {@macro setting}
+  Setting<int> intSetting({
+    required String key,
+    int defaultValue = 0,
+  }) {
+    final s = setting<int>(
```

Comment:
Shoot, sorry I missed this. Can we just consolidate the return to be direct? Thank you!

Reply:
Sorry about that, that's a relic from me debugging the compilation issue.